evo-delivery-app/evo-delivery-backend/src/services/driver.service.ts

```diff
@@ -1,24 +1,22 @@
 import { Driver } from '../types';
-import { DRIVERS } from './tmp-data';
+import driverModel, { IDriver } from '../database/models/driver.model';
 
-export const getDrivers = async (query, page, limit) => {
-  return DRIVERS as Driver[];
+export const getDrivers = async (filter: Object, page:number, limit:number) => {
+  return await driverModel.find({},null,{limit:limit});
```

Comment:
I think that the `await` keyword is redundant.
Because you wait for the promise and the then wrap it again with a promise (async function return value is always a promise)

Please check that.